theLudovyc/GUI/BuildingContainer.gd

```diff
@@ -8,15 +8,15 @@ extends VBoxContainer
 
 @onready var confirmation_dialog := %ConfirmationDialog
 
-const confirmation_text = "Are you sure you want to demolish this building?"
+const confirmation_text: String = "Are you sure you want to demolish this building?"
```

Comment:
You do not need to type a constant. Instead of variable they are not Variant. They are typed are at creation. https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#constants